pyproject.toml

```diff
@@ -13,8 +13,6 @@ authors = [
     {name = 'Eric Jolibois', email = 'em.jolibois@gmail.com'},
     {name = 'Hasan Ramezani', email = 'hasan.r67@gmail.com'},
 ]
-license-files = { paths = ['LICENSE'] }
```

Comment:
This is already matched by the PEP 639 defaults

Reply:
Well what we had before was not working, see https://pypi.org/project/dnserver/ for an example.

Reply:
That was with hatchling today.

Reply:
`license = {file = 'LICENSE'}` refers to the raw text, `license-files` is for metadata inclusion. adding `license = 'MIT'` now though, thought it was defined